apps/vr-tests-react-components/package.json

```diff
@@ -58,6 +58,7 @@
     "@fluentui/react-spinbutton": "*",
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30887&commitId=fe7e2a3a64c35313a2c2fa0b2d252e4e07da8508&env=prod&clientType=FLUENTUI)


  <details><summary><b>SwatchPicker Converged</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.Shape.default.chromium.png|0| Added|
|SwatchPicker Converged.Spacing.default.chromium.png|0| Added|
|SwatchPicker Converged.default - Dark Mode.default.chromium.png|0| Added|
|SwatchPicker Converged.default - High Contrast.default.chromium.png|0| Added|
|SwatchPicker Converged.default - RTL.default.chromium.png|0| Added|
|SwatchPicker Converged.default.default.chromium.png|0| Added|
|SwatchPicker Converged.spacing.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>